mayastor/tests/error_count.rs

```diff
@@ -35,12 +36,19 @@ static ERROR_DEVICE: &str = "error_device";
 static EE_ERROR_DEVICE: &str = "EE_error_device"; // The prefix is added by the vbdev_error module
 static BDEV_EE_ERROR_DEVICE: &str = "bdev:///EE_error_device";
 
+static YAML_CONFIG_FILE: &str = "/tmp/error_count_test_nexus.yaml";
+
 #[test]
 fn nexus_error_count_test() {
     common::truncate_file(DISKNAME1, 64 * 1024);
     common::truncate_file(DISKNAME2, 64 * 1024);
 
-    test_init!();
+    let mut config = Config::default();
+    config.err_store_opts.enable_err_store = true;
+    config.err_store_opts.err_store_size = 256;
```

Comment:
if not part of this commit then please ignore, but "num_entries" would be a more descriptive name.

Reply:
It wasn't part of this commit. But I'll consider renaming it in a future iteration.